breathe.gemspec

```diff
@@ -34,5 +34,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "pry", "~> 0.12.2"
   spec.add_development_dependency "webmock", "~> 3.7.6"
 
-  spec.add_dependency "faraday", "~> 0.17.0"
+  spec.add_dependency "sawyer", "~> 0.8.2"
```

Comment:
I don't know anything about Sawyer to be honest, but what made you want to switch and what problem does this solve?